### PR TITLE
feat: add ability to customize static error page.

### PR DESCRIPTION
**Pull request by b1ff**
Now customer can specify its own error page if default is not suitable. Also default error page now is a valid HTML and more pretty than just text.

### CODE REVIEW

File: ilc/server/errorHandler/ErrorHandler.js [hunk 1]
```diff
@@ -80,11 +83,34 @@ module.exports = class ErrorHandler {
...
+
+    #readStaticErrorPage(defaultContent) {
+        if (this.#staticFileContent != null) {
+            return this.#staticFileContent;
+        }
+
+        const staticFilePath = config.get('staticError.disasterFileContentPath');
+        try {
+            if (staticFilePath != null) {
+                this.#staticFileContent = readFileSync(staticFilePath).toString();
```
Comment (blackrabbit99) [inline ilc/server/errorHandler/ErrorHandler.js:106]: why sync ?
Maybe it makes sence serve it from memory ?

**Review by blackrabbit99 — CHANGES_REQUESTED** [review]

File: ilc/server/errorHandler/ErrorHandler.js (hunk 1 above)
Comment (b1ff) [reply ilc/server/errorHandler/ErrorHandler.js:106]: that is an idea. It is put into memory on first read

Comment (blackrabbit99) [reply ilc/server/errorHandler/ErrorHandler.js:106]: thank you

Comment (b1ff) [reply ilc/server/errorHandler/ErrorHandler.js:106]: you are welcome, can you approve PR?)

**Review by blackrabbit99 — APPROVED** [review]

**Pull request merged**